src/Compilers/Core/MSBuildTask/LocateEditorConfigFiles.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.CodeAnalysis.BuildTasks
+{
+    public sealed class LocateEditorConfigFiles : Task
```

Comment:
I had this impression too, but the Csc task is actually just called "Csc", not "CscTask".
https://github.com/dotnet/roslyn/blob/features/refout/src/Compilers/Core/MSBuildTask/Csc.cs